includes/polling/mempools/fiberhome-switch.inc.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * LibreNMS - FiberHome Switch device support - mempools module
+ *
+ * @category   Network_Monitoring
```

Comment:
Same for the copyright header as before.

Reply:
noted and done.